README.md

```diff
@@ -1,63 +1,107 @@
 
 Open source means more than open code. Open source provides equality for all and a chance for everyone to improve.
 
+
 ![Mautic](.github/readme_image.png "Mautic Open Source Marketing Automation")
 
-Get Involved
+Get Involved Quickly
 =============
-Before we tell you how to install and use Mautic, we like to shamelessly plug our awesome user and developer communities! Users, start [here][get-involved] for inspiration, or follow us on Twitter [@MauticCommunity][twitter] or Facebook [@MauticCommunity][facebook]. Once you’re familiar with using the software, maybe you will share your wisdom with others in our [Slack][slack] channel.
+Calling all devs, testers and tech writers! Get started quickly and show us what you’re made of. Technical contributions are also welcome.
+
+- 🐛 [Easy-to-fix bugs][t1-issues]
+- 🧪 [Best pull requests for testing][best-pulls]
 
-Calling all devs, testers and tech writers! Technical contributions are also welcome. First, read our [general guidelines][contributing] about contributing. If you want to contribute code, read our [CONTRIBUTING.md][contributing-md] or [Contributing Code][contribute-developer] docs then check out the issues with the [T1 label][t1-issues] to get stuck in quickly and show us what you’re made of.
+If you have questions, [check how to contribute](#-contribution).
+
+Using Mautic
+============
 
-If you have questions, the Mautic Community can help provide the answers.
+### ☁️ Deploy instantly with our Trials
 
-Installing and using Mautic
-============================
+[Request a trial][mautic-trials] filling up a form to deploy in minutes and test Mautic now.
 
-## Supported Versions
+### 🚀 Self-host completely free, forever
 
 Please check the latest supported versions on the [Mautic Releases](https://www.mautic.org/mautic-releases) page.
 
-## Software Downloads
 The GitHub version is recommended for both development and testing. The production package (including all libraries) is available at [mautic.org/download][download-mautic].
 
-## Installation
-### Disclaimer
-*Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download][download-mautic].*
+### Installation
+> [!WARNING]
+> <sub>*Install from source only if you are comfortable using the command line. You'll be required to use various CLI commands to get Mautic working and keep it working. If the source/database schema gets out of sync with Mautic releases, the release updater may not work and will require manual updates. For production, we recommend the pre-packaged Mautic which is available at [mautic.org/download][download-mautic].*
```

Comment:
I think we should only be saying they should install from source code if they are working locally. We don't support at all installing from Github in a prod environment. In that case we should be pointing them at the DDEV installation instructions, IMO. For Prod we should be pointing them at the Install with Composer documentation.

Reply:
Could you provide the new text? I'm afraid to do something wrong since I don't understand the technical thing